### PR TITLE
feat(shared): add security settings

### PR DESCRIPTION
**Pull request by ragi96**
[Task](https://app.clickup.com/t/8677unvhw) 

Security Settings if asset is download

added on
- audio-block
- image-block

### CODE REVIEW

File: packages/audio-block/src/settings.ts [hunk 1]
```diff
@@ -3,11 +3,14 @@
 import { FileExtensionSets } from '@frontify/app-bridge';
 import { AssetChooserObjectType, IconEnum, defineSettings } from '@frontify/guideline-blocks-settings';
 
-import { AudioSecurity, TextPosition } from './types';
+import { TextPosition } from './types';
+import {
+    getSecurityDownloadableSetting,
+    getSecurityGlobalControlSetting,
+} from '@frontify/guideline-blocks-shared/src/settings/security';
```
Comment (fulopdaniel) [inline packages/audio-block/src/settings.ts:10]:
```suggestion
} from '@frontify/guideline-blocks-shared'
```

File: packages/audio-block/src/AudioBlock.spec.ct.tsx [hunk 2]
```diff
@@ -133,4 +135,19 @@ describe('Audio Block', () => {
         mount(<AudioBlockWithStubs />);
         cy.get(AudioBlockSelector).should('have.css', 'flex-direction', 'column');
     });
+
+    it('renders an audio block without download button as its set in the security settings', () => {
```
Comment (fulopdaniel) [inline packages/audio-block/src/AudioBlock.spec.ct.tsx:139]:
```suggestion
    it('renders an audio block with download button as its set in the security settings', () => {
```

Comment (fulopdaniel) [reply packages/audio-block/src/AudioBlock.spec.ct.tsx:139]: Can you also add a test for not rendering it?

File: packages/image-block/src/ImageBlock.spec.ct.tsx [hunk 3]
```diff
@@ -50,8 +51,12 @@ describe('Image Block', () => {
         cy.get(ImageBlockImageSelector).should('exist');
     });
 
-    it('should render the download button if the image is uploaded', () => {
+    it('should render the download button if the image is uploaded and security settings allows it', () => {
```
Comment (fulopdaniel) [inline packages/image-block/src/ImageBlock.spec.ct.tsx:54]: Can you add an extra test case checking if the security does not allow it?

File: packages/audio-block/src/AudioBlock.spec.ct.tsx (hunk 2 above)
Comment (ragi96) [reply packages/audio-block/src/AudioBlock.spec.ct.tsx]: yep, added

File: packages/image-block/src/ImageBlock.spec.ct.tsx (hunk 3 above)
Comment (ragi96) [reply packages/image-block/src/ImageBlock.spec.ct.tsx:54]: also added one more test here

**Review by fulopdaniel — APPROVED** [review]

**Comment by fulopdaniel** [comment]
🥇

**Review by Walkeezy — APPROVED** [review]

**Pull request merged**